app/views/pageflow/entries/show.html.erb

```diff
@@ -33,3 +33,7 @@
 
   <%= render 'pageflow/entries/analytics', :entry => @entry %>
 <% end %>
+
+<% content_for :author, @entry.author %>
+<% content_for :publisher, @entry.publisher %>
+<% content_for :keywords, @entry.keywords %>
```

Comment:
Let's go for the breaking change and no longer  support [overriding `_meta_tags.html.erb](https://github.com/codevise/pageflow/wiki/Customizing-Views) now that we get a better mechanism.

Then we are free to move it to `pageflow/entries/_meta_tags.html.erb` render it from the `content_for(:head)` here and pass the entry explicitly as a local.